ballerina/modules/mock/service.bal

```diff
@@ -0,0 +1,527 @@
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+
+listener http:Listener ep0 = new (9090, config = {host: "localhost"});
+
+service / on ep0 {
```

Comment:
IMO we can start the service, only if the tests run against the mock server. for that, we can use the module init function to start the listener after the checking the relevant env variable.

Reply:
+1 changed with [71564eb](https://github.com/ballerina-platform/module-ballerinax-twilio/pull/218/commits/71564eb8b747b0315ff032df0c929b978247ffd4).